alicloud/provider.go

```diff
@@ -218,6 +218,7 @@ func Provider() terraform.ResourceProvider {
 			"alicloud_log_store_index":                     resourceAlicloudLogStoreIndex(),
 			"alicloud_log_machine_group":                   resourceAlicloudLogMachineGroup(),
 			"alicloud_logtail_config":                      resourceAlicloudLogtailConfig(),
+			"alicloud_logtail_to_machine_group":            resourceAlicloudLogtailToMachineGroup(),
```

Comment:
Please change `alicloud_logtail_to_machine_group` to `alicloud_logtail_attachment`